docs/docs/quickstart/install-guide.mdx

```diff
@@ -101,7 +101,7 @@ Changes made to a profile file may not apply until the next time you log into yo
 
 If you want to install Sedge on a Linux machine, we recommend installing some meta-packages to make the build process possible. You can install them with the following command in some of the major platforms:
 
-Ubuntu and Debian based: `sudo apt-get install gcc g++ build-essential`
+Ubuntu and Debian based: `sudo apt-get install gcc g++ abigen build-essential`
```

Comment:
Abigen is not directly possible to install using apt, can you add it as another dependency? or suggest the user to run a make command?

Reply:
When I used make install-abigen on WSL it didn't work, except after I used this command `sudo apt-get install gcc g++ abigen build-essential`. I would like to hear your recommendations.